partner_event/models/event_registration.py

```diff
@@ -11,6 +11,12 @@
 class EventRegistration(models.Model):
     _inherit = "event.registration"
 
+    attendee_partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Attendee Partner',
```

Comment:
🤔 I don't quite understand what's the field for?

Reply:
There was no `partner_id` in `event.registration` model before 9.0 in wich Odoo introduced. The problem is that Odoo's use is for billing purposes whereas that `event_partner` uses it to create or link each attendee to a different contact. So this field is to split those different behaviours.

Reply:
And what would happen if we use `partner_id` for this latter purpose and use the partner's billing data for bills?

I'd also thank some docs on this matter in the readme/code.

Reply:
I think that this question is decided by Odoo :smile: